x-pack/plugins/screenshotting/server/screenshots/get_screenshots.ts

```diff
@@ -52,10 +54,13 @@ export const getScreenshots = async (
       description: item.attributes.description,
     });
 
-    span?.end();
+    eventLogger.getScreenshotEnd({
```

Comment:
Shall we also handle the case when "get screenshots" fails?

Reply:
Added in d5a83bc75efa